observations/snow/adpsfc_snow.yaml.j2

```diff
@@ -0,0 +1,110 @@
+-
+
+  # Observation Space (I/O)
+  # -----------------------
+  obs space:
+    name: adpsfc_snow
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: "{{snow_obsdatain_path}}/{{snow_obsdatain_prefix}}{{observation_from_jcb}}{{snow_obsdatain_suffix}}"
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: "{{snow_obsdataout_path}}/{{snow_obsdataout_prefix}}{{observation_from_jcb}}{{snow_obsdataout_suffix}}"
+    simulated variables: [totalSnowDepth]
+  #
+
+  # Observation Operator
+  # --------------------
+  obs operator:
+    name: Composite
+    components:
+    - name: Identity
+    - name: BackgroundErrorIdentity
+  obs error:
+    covariance model: diagonal
+
+  # Observation Filters (QC)
+  # ------------------------
+  obs pre filters:
+    - filter: Perform Action
+      filter variables:
+      - name: totalSnowDepth
+      action:
+        name: assign error
+        error parameter: 40.0
+    - filter: Variable Assignment
+      assignments:
+      - name: GrossErrorProbability/totalSnowDepth
+        type: float
+        value: 0.02
+      - name: BkgError/totalSnowDepth_background_error
+        type: float
+        value: 30.0
+  obs prior filters:
+    - filter: Bounds Check
+      filter variables:
+      - name: totalSnowDepth
+      minvalue: 0.0
+      maxvalue: 2000.0
+      action:
+        name: reject
+    - filter: Domain Check
+      where:
+      - variable:
+          name: MetaData/stationElevation
+        minvalue: -999.0
+    - filter: Domain Check # land only
+      where:
+      - variable:
+          name: GeoVaLs/slmsk
+        minvalue: 0.5
+        maxvalue: 1.5
+    - filter: Domain Check  # no land-ice
```

Comment:
This should be  `- filter: RejectList # land only`. Otherwise, only the observations over land-ice will be kept, but all other observations over land will be rejected.